x-pack/plugins/case/common/api/cases/configure.ts

```diff
@@ -31,9 +32,9 @@ const CaseFieldRT = rt.union([
 ]);
 
 const ThirdPartyFieldRT = rt.union([
-  JiraFieldsRT,
-  ServiceNowFieldsRT,
-  ResilientFieldsRT,
+  JiraCaseFieldsRt,
```

Comment:
nit -> I am wondering if `JiraConnectorFields` is more descriptive. what do you think?